src/V3ActiveTop.cpp

```diff
@@ -77,6 +77,21 @@ class ActiveTopVisitor final : public VNVisitor {
             VL_DO_DANGLING(nodep->unlinkFrBack()->deleteTree(), nodep);
             return;
         }
+
+        // Delete empty procedures
+        for (AstNode *stmtp = nodep->stmtsp(), *nextp; stmtp; stmtp = nextp) {
+            nextp = stmtp->nextp();
+            if (AstNodeProcedure* const procp = VN_CAST(stmtp, NodeProcedure)) {
+                if (!procp->stmtsp()) VL_DO_DANGLING(procp->unlinkFrBack()->deleteTree(), stmtp);
```

Comment:
```suggestion
                if (!procp->stmtsp()) VL_DO_DANGLING(pushDeletep(procp->unlinkFrBack()), stmtp);
```

Makes it easier to debug as new nodes won't get new'ed at old proc's address.  Similar below.